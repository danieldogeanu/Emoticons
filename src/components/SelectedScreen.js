import React from 'react';
import './SelectedScreen.scss';

function SelectedScreen() {
	return (
		<div className="SelectedScreen"></div>
	);
}

export default SelectedScreen;