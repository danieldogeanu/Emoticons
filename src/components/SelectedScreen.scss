@import '../styles/includes';

.SelectedScreen {
	display: none;
	justify-content: center;
	align-items: center;
	position: fixed;
	z-index: 1000;
	top: 0; left: 0;
	bottom: 0; right: 0;
	margin: 0; padding: 0;
	font-size: rem(180);
	line-height: 1;
	opacity: 0.7;
	user-select: none;
	@include breakpoint(min, rem(481)) { @include fluid-type(481, 1200, 180, 400); }
	@include breakpoint(min, rem(1201)) { font-size: rem(400); }
	@include breakpoint-orientation(max, rem(600), landscape, height) { font-size: 50vh; }
	&.show { display: flex; }
}