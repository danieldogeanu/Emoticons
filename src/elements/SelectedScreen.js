import React from 'react';
import '../styles/components/SelectedScreen.scss';

const SelectedScreen = () => (<div className="SelectedScreen" data-testid="SelectedScreen"></div>);

export default SelectedScreen;
