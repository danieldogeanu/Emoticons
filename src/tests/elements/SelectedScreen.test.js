import React from 'react';
import {render} from '@testing-library/react';
import SelectedScreen from '../../elements/SelectedScreen';

describe('SelectedScreen Element', () => {

	it('renders selected screen properly', () => {
		const {getByTestId} = render(<SelectedScreen />);
		const renderedScreen = getByTestId('SelectedScreen');

		expect(renderedScreen).toBeInTheDocument();
		expect(renderedScreen).toHaveClass('SelectedScreen');
		expect(renderedScreen).toMatchSnapshot();
	});

});
