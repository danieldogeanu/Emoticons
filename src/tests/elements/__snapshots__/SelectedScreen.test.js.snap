// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SelectedScreen Element renders selected screen properly 1`] = `
<div
  class="SelectedScreen"
  data-testid="SelectedScreen"
/>
`;
